packages/volto/src/start-server.js

```diff
@@ -19,7 +19,7 @@ export default function server() {
   server
     .listen(port, bind_address, () => {
       if (app.apiPath === app.publicURL || !app.apiPath) {
-        console.log(`Volto is running in SEAMLESS mode`);
+        console.log(`Volto is running in normal deployment mode`);
       } else {
         console.log(`API server (API_PATH) is set to: ${app.apiPath}`);
```

Comment:
Maybe it would be clearer to remove the condition altogether and always log line 24?

Reply:
@davisagli remove the `else` condition only, or both the `if` and `else` conditions?

Reply:
Yeah, I thought that the condition might be useful, but it's clear that it's not.